core-bundle/tests/Contao/StringUtilTest.php

```diff
@@ -218,12 +218,6 @@ public function parseSimpleTokensProvider(): \Generator
             '{if token=="foo"}',
         ];
 
-        yield 'Test nested if-tag with " in value (match)' => [
-            '{if value=="f"oo"}1{endif}{if value=="f\"oo"}2{endif}',
```

Comment:
The test for escaping, i.e.
```
{if value=="f\"oo"}2{endif}
``` 
should stay though :)

Reply:
Added in ea5a60400ecef48318ac824109c09e7db183fc2f.